llama-index-integrations/llms/llama-index-llms-cerebras/pyproject.toml

```diff
@@ -30,12 +30,12 @@ license = "MIT"
 name = "llama-index-llms-cerebras"
 packages = [{include = "llama_index/"}]
 readme = "README.md"
-version = "0.2.0"
+version = "0.2.1"
 
 [tool.poetry.dependencies]
 python = ">=3.9,<4.0"
 llama-index-core = "^0.12.0"
-llama-index-llms-openai-like = "^0.3.0"
+llama-index-llms-openai-like = "^0.3.1"
```

Comment:
I bumped and published openai and openai-like locally, then double bumped it so that gha could publish everything in one swoop for me :)